WIP-SQL/connection-strings.md

```diff
@@ -0,0 +1,111 @@
+# Connection strings
+author: SebaRaba
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[More on connection strings](http://www.dofactory.com/reference/connection-strings)'
+
+---
+## Content
+
+The syntax for conneciton strings is slighty different for all types of servers. There are two main connection types:
+- ODBC (Open Data Base Connectivity)
+- OLE DB (Object Linking Embedding)
+
+**ODBC** is a connection method to data sources. It requires  a data source (DSN) to be set up. This can be done using a *SQL* driver or other drivers if you want to connect to a different type of database. It's designed for connecting relational databases.
```

Comment:
"It requires a data source (DSN) to be set up" - this is I believe a reference to how Microsoft stacks do it. Remove this sentence.

What you really mean is, it requires a remote database to exist somewhere, and you need to know where it is.
A connection string is simply a URL pointing at a database that is hosted on a server, locally or remotely. 

Here's an example:

postgres://admin:password@localhost:5432/pokemon
The different parts:
protocol (postgres://, mysql://, mssql://, depending on the database engine)
username (admin)
password (password)
hostname (localhost)
port (5432)
database name (pokemon)